update the links to GitHub Discussions

yin1999 opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

update the links to GitHub Discussions (organization-wide)

yin1999 committed:
update the links to GitHub Discussions

Josh-Cena reviewed (commented):
If we are to do this change, could you update the other occurences of https://github.com/mdn/mdn-community/discussions as well? I don't have preferences for either, although the fact that https://github.com/mdn/mdn-community is used as the org-wide discussions repo does sound like a changeable implementation detail to me, so this is reasonable.

yin1999 committed:
chore: replace all links to "mdn/mdn-community"

yin1999 commented:
> If we are to do this change, could you update the other occurences of https://github.com/mdn/mdn-community/discussions as well? 

I've replaced all the occurences (and modified the content to match [the current categories](https://github.com/orgs/mdn/discussions).

> I don't have preferences for either, although the fact that https://github.com/mdn/mdn-community is used as the org-wide discussions repo does sound like a changeable implementation detail to me, so this is reasonable.

As those links to mdn/mdn-community discussions have been redirected to organization-wide discussions. And the redirection would break the link to specific category. Maybe it's a better idea to update them (reducing redirection and could follow the link to specific categories).

yin1999 committed:
Merge branch 'main' into patch-2

Josh-Cena commented on files/en-us/mdn/community/discussions/index.md:
```diff
@@ -57,41 +57,30 @@ Check out the definition of each discussion category below so that you can start
-        <a href="https://github.com/mdn/content">content</a> on MDN Web Docs.
+        Discussions related to improvements to existing
+        <a href="https://github.com/mdn/content">content</a>,
+        ideas for new content, or any other content suggestions on MDN Web Docs.
```
New content requests should go to https://github.com/mdn/mdn. So you should update files/en-us/mdn/writing_guidelines/what_we_write/index.md as well.

yin1999 replied:
Do you mean we need to create an issue in [mdn/mdn](https://github.com/mdn/mdn/issues) to place a new content request (so I need to remove the description about "new content" here)?

Josh-Cena replied:
Yes, and I guess that's why the "content request" category doesn't exist any more.

yin1999 committed:
remove description about new content seggestion in Discussions

Josh-Cena reviewed (approved):
Great. Thanks!
